cmd/grype/cli/commands/db_check.go

```diff
@@ -30,6 +30,8 @@ func DBCheck(app clio.Application) *cobra.Command {
 }
 
 func runDBCheck(opts options.Database) error {
+	// `grype db check` should _always_ check for updates, regardless of config
+	opts.MinAgeToCheckForUpdate = 0
```

Comment:
I thought about adding a separate bool "force check" option, or a force check parameter to IsUpdateAvailable and Update. Would that have been preferable?

Reply:
I feel that not changing the behavior of db check is the right call for now from an options-perspective. We can always add more config items later to opt into `force`.

But, this param should be changed within `PreRunE` such that the config options are finalized before logging them.